kolibri/utils/server.py

```diff
@@ -100,6 +103,19 @@ def run(self):
         call_command("ping")
 
 
+class VacuumThread(threading.Thread):
+
+    @classmethod
+    def start_command(cls):
+        thread = cls()
+        thread.daemon = True
+        thread.start()
+
+    def run(self):
+        # Try to do the vacuum every 3 hours
+        call_command("vacuumsqlite", interval=180)
```

Comment:
In order to ensure that this gets thoroughly tested internally, we could set the interval to something much more aggressive like 10 minutes. Then we'd simply plan on increasing the interval before publicly releasing. Thoughts?

Reply:
That's a great idea @indirectlylit !

Reply:
@benjaoming  you said in a previous comment you are against of running vacuum periodically, or maybe I misunderstood you?

Reply:
No, see my previous comment - I just don't get where the job should be controlled from. If we are doing this in its own thread, let's stick to that and make the behavior configurable and possible to disable.

Reply:
For instance, it shouldn't run on Postgres.

Reply:
Ah, okay, the thread just exits in other db environments - but for instance, Postgres does actually have a VACUUM command :)

https://www.postgresql.org/docs/9.5/static/sql-vacuum.html

Reply:
Yes, I know it (believe me, do never run that command in a postgresql database in production, setting autovacuum is much better for postgresql), but if you check the code you will see the vacuumsqlite command checks if the db is sqlite, if not, it just does nothing, so it does not run on Postgresql

Reply:
@indirectlylit what do you think about the way to alter the interval for testing? should we hardcode it or create a new parameter in options.ini?

Reply:
the easiest thing to do is just hard-code it, and open up an issue targeted to 0.11.0 reminding us to change it back before release :)